### ISSUE TITLE
Canvas performance degrades significantly when using Minimal Theme vs Default Theme

### ISSUE THREAD

**Issue by nickchomey** [kepano/obsidian-minimal#516]
### Is the bug present when using the default Obsidian theme?

Yes

### Is the bug present when snippets and plugins are disabled?

Yes

### Minimal theme version

6.1.17

### Describe the bug

I just found that with Minimal Theme turned on, Canvas performance degrades significantly. I started a discussion in insider-desktop here: https://discord.com/channels/686053708261228577/716028884885307432/1054935468506103818 and they said to report it directly to Minimal

At first, I had a canvas with a bunch of embedded note files. I thought that perhaps the performance degradation was from having 40,000+ words in all of those embedded notes, so I replaced all the Note cards with Canvas cards that contain simple `[[link to note]]` links. It helped, but it was still sluggish - particularly when zooming in and out. 

When I switch to the default theme, it is snappy again. If I copy and paste all of the cards such that there's 4x more cards, its still reasonably snappy with default and completely unusable with Minimal. I'm happy to try to give some diagnostic info if you can guide me!

### Debug info

SYSTEM INFO:
	Obsidian version: v1.1.8
	Installer version: v0.15.9
	Operating system: Windows 10 Home 10.0.22623
	Login status: logged in
	Catalyst license: insider
	Insider build toggle: on
	Live preview: on
	Legacy editor: off
	Base theme: dark
	Community theme: Minimal v6.1.17
	Snippets enabled: 2
	Restricted mode: on

RECOMMENDATIONS:
	Custom theme and snippets: for cosmetic issues, please first try updating your theme and disabling your snippets. If still not fixed, please try to make the issue happen in the Sandbox Vault or disable community theme and snippets.

**Comment by kepano** [comment]
I'm not able to reproduce this. Can you share a .canvas file where the issue is present?

**Comment by kepano** [comment]
Also can you share your device specs?

**Comment by icebruce** [comment]
I am observing a similar issue. Without using any emended, just creating a bigger number of cards and the canvas stars lagging slightly when panning and zooming.
[Untitled.md](https://github.com/kepano/obsidian-minimal/files/10305141/Untitled.md)
**Note:** cannot attach .canvas files so I changed the extension to .md

Tested by disabling all plugins and css snippets, still the same. Using same file on a vault with default theme does not present the same issue.

https://user-images.githubusercontent.com/66655064/209592287-91406a5a-2142-4725-bc10-0d3a1011ebc0.mp4

SYSTEM INFO:
	Obsidian version: v1.1.9
	Installer version: v1.1.9
	Operating system: Windows 10 Pro 10.0.19045
	Login status: logged in
	Catalyst license: insider
	Insider build toggle: on
	Live preview: on
	Legacy editor: off
	Base theme: dark
	Community theme: Minimal v6.1.17
	Snippets enabled: 7
	Restricted mode: off
	Plugins installed: 33
